images/nginx/rootfs/build.sh

```diff
@@ -54,8 +54,10 @@ export LUA_RESTY_LOCK=0.08
 export LUA_RESTY_UPLOAD_VERSION=0.10
 export LUA_RESTY_STRING_VERSION=0.12
 
-export OPENSSL_VERSION=1.0.2u
-export OPENSSL_FIPS_VERSION=2.0.16
+export CMAKE_VERSION=3.18.3
+export LIBMAXMINDDB_VERSION=1.4.3
```

Comment:
Just want to confirm, where you found that version ? Is this version compatible with geoip2 nginx module ?